shell/const.go

```diff
@@ -0,0 +1,8 @@
+// +build !windows
+
+package shell
+
+const (
+	ENVIRONMENT_KEY_PATH      = "PATH"
```

Comment:
These should be `EnvironmentKeyPath` if your following golang style. Snakes don't live here! :snake: :no_entry_sign: